lib/node-debugger.coffee

```diff
@@ -26,6 +26,9 @@ module.exports =
     nodeArgs:
       type: 'string'
       default: ''
+    scriptMain:
```

Comment:
renamed setting to script main.  to closer resemble the property in a package.json file